packages/core/README.md

```diff
@@ -2,9 +2,9 @@
 
 # [Blueprint](http://blueprintjs.com/) Core Components
 
-Blueprint is a React UI toolkit for the Web.
+Blueprint is a React UI toolkit for the web.
 
-This package contains the core set of UI components as CSS and React code.
+This package contains the core set of UI components as Sass and React code.
```

Comment:
this isn't really accurate because it shows up on https://www.npmjs.com/package/@blueprintjs/core and that's actually where I expect most people to see it, not in this Github repo